components/filter/filter.js

```diff
@@ -276,6 +289,40 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		`;
 	}
 
+	_clearDimension(dimension) {
+		switch (dimension.type) {
+			case 'd2l-filter-dimension-set': {
+				this.shadowRoot.querySelector(`#${SET_DIMENSION_ID_PREFIX}${dimension.key}`).setSelectionForAll(false);
+				break;
+			}
+		}
+	}
+
+	_clearDimensionData(dimension, sendClearEvent, sendIndividualEvents) {
```

Comment:
This function does things that aren't utilized in this PR, but are in the Clear All PR: https://github.com/BrightspaceUI/core/pull/1635/files#diff-371eee884788627e4694af0044b256de9257505f6ee6430c4302574842f8b441R474-R478